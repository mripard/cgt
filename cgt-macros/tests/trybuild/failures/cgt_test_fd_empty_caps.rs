#[cgt_macros::cgt_test_with_fd(capabilities)]
fn test(_: std::os::fd::BorrowedFd<'_>) -> Result<(), cgt_core::TestError> {
    Ok(())
}

fn main() {}