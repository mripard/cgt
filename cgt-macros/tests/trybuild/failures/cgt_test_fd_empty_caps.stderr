error: unexpected end of input, expected assignment `=`
 --> tests/trybuild/failures/cgt_test_fd_empty_caps.rs:1:1
  |
1 | #[cgt_macros::cgt_test_with_fd(capabilities)]
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the attribute macro `cgt_macros::cgt_test_with_fd` (in Nightly builds, run with -Z macro-backtrace for more info)
